src/main/kotlin/no/fdk/dataset_catalog/utils/ZonedDateTimeDeserializer.kt

```diff
@@ -0,0 +1,25 @@
+package no.fdk.dataset_catalog.utils
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import java.time.*
+import java.time.format.DateTimeFormatter
+
+
+class ZonedDateTimeDeserializer : JsonDeserializer<ZonedDateTime>() {
```

Comment:
Dette er overkill. Hvorfor ikke bruke `LocalDateTime` som har ferdige serialisere/deserialisere?

Reply:
Tror det bare finnes en ZonedDateTimeSerializer, og ikke ZonedDateTimeDeserializer. Men det betry kanskje at det fungerer å bruke DateTimeDeserializer?

Reply:
Stemmer, det finnes ingen deserializer for ZonedDateTime akkurat nå, så implementerte denne for å støtte vårt bruk. Finner ingen klasse som heter DateTimeDeserializer @NilsOveTen ?